perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -938,6 +940,50 @@ public void getAssignedMembers() throws Exception {
 
 	}
 
+	@Test
+	public void getAssignedMembersWithStatus() throws Exception {
+		System.out.println(CLASS_NAME + "getAssignedMembersWithStatus");
+
+		Vo vo = setUpVo();
+
+		Group group1 = perun.getGroupsManager().createGroup(sess, vo, new Group("group1", "group1")); // active
+		Group group2 = perun.getGroupsManager().createGroup(sess, vo, new Group("group2", "group2")); // inactive
+		Group group3 = perun.getGroupsManager().createGroup(sess, vo, new Group("group3", "group3")); // not assigned
+
+		Member member1 = setUpMember(vo);
+		Member member2 = setUpMember(vo);
+		Member notAssignedMember = setUpMember(vo);
+
+		perun.getGroupsManager().addMember(sess, group1, member1);
+		perun.getGroupsManager().addMember(sess, group2, member1);
+		perun.getGroupsManager().addMember(sess, group3, member1);
+		perun.getGroupsManager().addMember(sess, group2, member2);
+		perun.getGroupsManager().addMember(sess, group3, notAssignedMember);
+
+		Facility facility = new Facility(0, "testFacility", "testFacility");
+		facility = perun.getFacilitiesManagerBl().createFacility(sess, facility);
+		Resource resource = new Resource(0, "testResource", "testResource", facility.getId(), vo.getId());
+		resource = perun.getResourcesManagerBl().createResource(sess, resource, vo, facility);
```

Comment:
You can probably use _setUpFacility()_ and _setUpResource()_ methods here.

Reply:
good point